testing_exercise/testing.js

```diff
@@ -0,0 +1,55 @@
+function replaceWith(str, char1, char2) {
+  var newStr = "";
+
```

Comment:
Try to start being consistent with your spacing after defining variables. No new lines after each one.